tests/unit/test_routes.py

```diff
@@ -594,6 +594,47 @@ def test_search(module_client, module_db, fake_auth_from_oc, fake_algolia_save,
     assert (result.json['data'][0]['url'] == resource.json['data'].get('url'))
 
 
+def test_search_paid_filter(module_client, module_db, fake_auth_from_oc, fake_algolia_save, fake_algolia_search):
+    client = module_client
+
+    apikey = get_api_key(client)
+
+    # Create paid and unpaid resource to find it in the search results.
```

Comment:
Since we're mocking the search results with `fake_algolia_search` there shouldn't need to be anything in the database. We can probably save the cycles and just take this whole thing out.

Reply:
That being said, if you can make a case for why this is necessary for this test, let me know and I'll hear you out.